sign in page

deba-saha opened the pull request:
this is the sign in page allowing users to sign in using email/username and password or with facebook.

deba-saha committed:
sign in page

this is the sign in page allowing users to sign in using email/username and password or with facebook.

deba-saha committed:
fixed facebook button

fixed the sign in with facebook button by adding the facebook logo and fixing the colour of the button to match the icons colour

deba-saha committed:
added sign in with google button

added the option to allow users to sign in with google. added the google logo and changed the colour of the button to match thee logos colour

PaulMicu reviewed (changes requested):
you need to clean the index file before merging

deba-saha committed:
fixing the sign in php file

i was working on the index php file and never updated the actual sign in php and there were problems merging it

Commit:
fixed index and header

reverted accidental changes to those files
